### PR TITLE
Upgrade Child Theme to version 2

### PR DESCRIPTION
**Pull request by matt-bernhardt**
(Edit) This has turned into a massive pull request that updates nearly the entirety of the Child theme, and when merged will become version 2 of that theme.

### CODE REVIEW

**Comment by PBruk** [comment]
@matt-bernhardt can you please do a review of my PR for this branch? Apologies, in advance, for the volume of commits...

File: page-full-width.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ Template Name: Full Width
```
Comment (matt-bernhardt) [inline page-full-width.php:3]: This line needs an asterisk before the text starts

**Comment by matt-bernhardt** [comment]
I'm reviewing the changes now, and will add comments where appropriate. This is the version running on Dev, correct?

File: style.css [hunk 2]
```diff
@@ -1,265 +1,275 @@
+@import "../libraries/style.css";
```
Comment (matt-bernhardt) [inline style.css]: The Wordpress Codex talks about parent themes being loaded via functions.php - see MITLibraries/courtyard-blog#23 for more info

**Comment by PBruk** [comment]
This version is running on Dev, yes.

**Comment by matt-bernhardt** [comment]
Are you still piecing together changes with more to come, or is this all of them? I ask because I don't see any changes to functions.php yet - and I thought there would be to define widget regions.

**Comment by PBruk** [comment]
This is an ongoing process. As far as widgets go, I am inheriting the widget function from Parent and I am calling the function in the - new - content-widgetized.php file, which itself is being called by the - new - page-widgetized.php file.

Probably, tho, once we decide what we want those areas to be titled, on the backend, since they are currently just first front page widget area and second front page widget area, we should add the function to the functions.php in Child and rename it there.

**Comment by matt-bernhardt** [comment]
Yeah, the way things are called now is producing duplicate ID values (both regions have `id="secondary"`) which isn't valid HTML - and may screw up styling attempts down the road.

I'd recommend sorting out what the regions are going to be called, and building out the relevant functions, sooner rather than later.

**Comment by PBruk** [comment]
Okay, gotcha... I'll jump on that next'ish. Thanks for the head's up. Didn't see that!

File: category-maihaugen-gallery.php [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+			       
+			       <?php } ?>
+		       
+		       <?php wp_reset_query(); // Restore global post data stomped by the_post().
+
+		endif; ?>
+			   		   
+			</div>
+
+		<!---- END OF CURRENT EXHIBITS LOOP --->
```
Comment (frrrances) [inline category-maihaugen-gallery.php]: extra dashes here

File: category-maihaugen-gallery.php [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+				<?php } ?>       
+    
+		<?php wp_reset_query(); // Restore global post data stomped by the_post().
+
+		endif; ?>
+		
+			</div>
+		
+		
+		<!---- END OF UPCOMING EXHIBITS LOOP ----->
```
Comment (frrrances) [inline category-maihaugen-gallery.php]: extra dashes here

File: category-maihaugen-gallery.php [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+		       get_template_part( 'inc/exhibits-past' );
+
+		       wp_reset_query(); // Restore global post data stomped by the_post().
+
+			   endwhile; // end of the loop. ?>
+
+			</div>
+
+
+		<!--- END OF UPCOMING EXHIBITS LOOP --->
```
Comment (frrrances) [inline category-maihaugen-gallery.php]: extra dash

**Comment by matt-bernhardt** [comment]
After plenty of work and testing, this branch appears to be ready to merge. Cross your fingers...

**Pull request merged**